vnavmesh/SceneExtractor.cs

```diff
@@ -187,6 +187,38 @@ private unsafe Mesh AddMesh(string path, MeshType type)
         return mesh;
     }
 
+    internal Mesh UnwalkableMesh(Mesh mesh)
```

Comment:
Why would you need this mesh cloning? It should be enough to set ForceSetPrimFlags in instances of the mesh, without changing actual primitive data.

Reply:
Well I did this simply because all instances needed to be marked as unwalkable but if you would prefer I do it to each instance instead of globally I will change that tonight after the draft

Reply:
This has been completed!! thanks